Documentation/source/guides/getting-started.rst

```diff
@@ -105,12 +106,13 @@ An example of a username created with this configuration would be ``550e8400-e29
         var giniSDK: GiniSDK?
 
         func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-            
+
             self.giniSDK = GINISDKBuilder.anonymousUserWithClientID("your_gini_client_id", clientSecret: "your_gini_client_secret", userEmailDomain: "example.com").build()
+            // It is possible to enable certificate pinning adding the 'certificatePath' in the `GINISDKBuilder` initializer.
```

Comment:
I don't understand why you have this comment twice. Also, either show the code as well, or remove the comment in OMO.